go-controller/pkg/libovsdb/util/northd_sync.go

```diff
@@ -42,24 +42,31 @@ func WaitUntilNorthdSyncOnce(ctx context.Context, nbClient, sbClient client.Clie
 	if err != nil {
 		return fmt.Errorf("failed to generate ops to mutate nb_cfg: %w", err)
 	}
-	expectedNbCfgValue := nbGlobal.NbCfg + 1
 	if _, err = libovsdbops.TransactAndCheck(nbClient, ops); err != nil {
 		return fmt.Errorf("failed to transact to increment nb_cfg: %w", err)
 	}
-	sbGlobal := &sbdb.SBGlobal{}
-	// poll until we see the expected value in SB DB every 5 milliseconds until context is cancelled.
+	expectedNbCfgValue := nbGlobal.NbCfg + 1
+	if expectedNbCfgValue < 0 { // handle overflow
+		expectedNbCfgValue = 0
+	}
+	nbGlobal = &nbdb.NBGlobal{}
+	// ovn-northd sets hv_cfg to the smallest sequence number of all the chassis in the system,
+	// as reported in the Chassis_Private table in the southbound database. Thus, hv_cfg
```

Comment:
and for IC this is just 1 chassis per node/zone right?

Reply:
yep - if we assume 1 node per zone. Ill change it to that.